### PR TITLE
 challenge done

### PR DESCRIPTION
**Pull request by curriejames**
challenge done

### CODE REVIEW

File: James/book_class.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Booklist():
+	def __init__(self):
+		self.books = []
+
+
+	def add(self, title, author):
```
Comment (Tyler-Toro) [inline James/book_class.py:6]: When you run these files, there is an error code with this function. Were you able to see this?

The add function should create an empty dictionary 'book' , set dictionary keys 'title' and 'author' to the method attributes, then append as the last step

def add(self, title, author)
   book = {}
   book['title'] = title
   book['author'] = author

   self.books.append(book)

File: James/book_challenge.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Just Mercy - Bryan Stevenson
+The New Jim Crow - Michelle Alexander
+The Truths We Hold - Kamala Harris
+My Grandmother's Hands - Resmaa Menakem
+
+Finally, prinb the books attribute of my_library to make sure your books have been added!
+'''
+
+print('\nPart 2\n')
+my_library.add( title ='Just Mercy', authors='Bryan Stevenson')
```
Comment (Tyler-Toro) [inline James/book_challenge.py:48]: This add function is correctly formatted here, but does not work because the add function is not correct

**Review by Tyler-Toro — APPROVED** [review]
Approving so files can be added to your branch. BUT these need to be revisited and corrected with another pull request later on

**Comment by curriejames** [comment]
Yes I saw the error I needed help but I wanted to make sure I was on time with my challenges

**Comment by Tyler-Toro** [comment]
Never worry about time. I want you to solve these problems and struggle
with them vs sending them in a day or two on time.

**Pull request merged**